controllers/config/operator_constants.go

```diff
@@ -4,6 +4,7 @@ const (
 	// Grafana
 	GrafanaImage   = "docker.io/grafana/grafana"
 	GrafanaVersion = "9.1.6"
+	GrafanaDigest  = "sha256:ff68ed4324e471ffa269aa5308cdcf12276ef2d5a660daea95db9d629a32a7d8"
```

Comment:
The readability is low and it's not needed in a disconnected env since you can overwrite it.
Please remove